go/client/cmd_paperprovision.go

```diff
@@ -23,16 +23,23 @@ func NewCmdPaperProvision(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cl
 		Action: func(c *cli.Context) {
 			cl.ChooseCommand(NewCmdPaperProvisionRunner(g), "paperprovision", c)
 		},
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "keep-paper-key, k",
```

Comment:
well as long as you think there's nothing else depending on the existing behavior.  i'm just always worried i'm going to break something so try to always maintain backwards compatibility, that's my bias.